uk.co.compucorp.civicrm.hrcore/CRM/HRCore/Test/Fabricator/Contact.php

```diff
@@ -21,4 +21,14 @@ public static function fabricate($params = []) {
 
     return array_shift($result['values']);
   }
+
+  public static function fabricateWithEmail($params, $email) {
+    $contact = self::fabricate($params);
```

Comment:
Is this method really necessary? Did you try just pass the e-mail among the params? Like this: `fabricate(['name' => 'bla bla bla', 'email' => 'bla@blabla.bla'])`